### PR TITLE
add support for midi velocity

### PR DESCRIPTION
**Pull request by rettinghaus**
This PR addresses #1109.
It adds Verovios default midi values to `options.h` and sets default velocity to default MIDI 1.0 value (which is 90). 
Additional support for r/w of `@vel` on `<note>` and MIDI playback.

### CODE REVIEW

File: include/vrv/options.h [hunk 1]
```diff
@@ -35,6 +35,13 @@ class OptionGrp;
 #define MIN_SCALE 1
 #define MAX_SCALE 1000
 
+//----------------------------------------------------------------------------
```
Comment (lpugin) [inline include/vrv/options.h]: Please move these to vrvdef.h

Comment (rettinghaus) [reply include/vrv/options.h]: Didn't put them there, but ok.

Comment (lpugin) [reply include/vrv/options.h]: I meant the midi values

Comment (rettinghaus) [reply include/vrv/options.h]: As you wish …

**Pull request merged**